spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -477,9 +496,7 @@ public void consumeFromAnEmbeddedTopic(Consumer<?, ?> consumer, String topic) th
 	 * @throws Exception an exception.
 	 */
 	public void consumeFromEmbeddedTopics(Consumer<?, ?> consumer, String... topics) throws Exception {
-		for (String topic : topics) {
-			assertThat(this.topics).as("topic '" + topic + "' is not in embedded topic list").contains(topic);
-		}
+		assertThat(this.topics).as("topic(s):'" + Arrays.toString(topics) + "' are not in embedded topic list").containsAll(new HashSet<>(Arrays.asList(topics)));
```

Comment:
Well. I would prefer to assert all, but report only wrong topics - don't tell that the whole set is wrong.